db/models.py

```diff
@@ -1 +1,48 @@
 from django.db import models
+
+
+class Race(models.Model):
+    names = [
+        ("elf", "Elf"),
```

Comment:
not all races are included.Also it is a convention to call Choices options with capital letters and add _CHOICES to the end to distinguish from regular fields